cypress/integration/demo_tests/demo_spec.js

```diff
@@ -60,4 +60,66 @@ describe('opening OpenSearch-Dashboards', () => {
         cy.get('[data-attr-field="category"]').click()
         cy.get('[data-test-subj="fieldToggle-category"]').click()
     })
+
+    it('Explore the data in the dashboard', () => {
+        /**
+         * Known issue: Comboboxes UI elements are behaving inconsistently,
+         * can only sometimes successfully "click on" and select options.
+        **/
+        cy.get('[data-test-subj="toggleNavButton"]').click()
+        // Click the Dashboards button in the Nav menu
+        cy.get('[data-test-subj="collapsibleNavAppLink"]').contains('Dashboard').click()
+
+        // Check if automatically opening a dashboard or not
+        cy.url().then((path) => {
+            if (path.includes('/app/dashboards#/list')) {
+                cy.get('[data-test-subj="dashboardListingTitleLink-[eCommerce]-Revenue-Dashboard"]').click()
+            }
+        })
+        // Make sure the desired UI element is visible
+        cy.get('[data-test-subj="inputControl0"]').should('be.visible')
+
+        /**
+         * Known issue: when Combobox elements are already selected,
+         * cannot remove just using a text clear, will need to implement
+         * a conditional check to click the "clear options" button if it exists
+         **/
+        // Select "Gnomehouse"
+        cy.get('[data-test-subj="listControlSelect0"]').find('[data-test-subj="comboBoxInput"]').type('{selectall}{backspace}Gnomehouse')
+        cy.get('[class="euiFilterSelectItem"]').should('have.length', 2)
+        cy.get('[data-test-subj="listControlSelect0"]').find('[data-test-subj="comboBoxInput"]').focus()
+        cy.get('[title="Gnomehouse"]').click({ force: true })
+
+        // Select "Women's Clothing"
+        cy.get('[data-test-subj="listControlSelect1"]').find('[data-test-subj="comboBoxInput"]').type("{selectall}Women's Clothing", { timeout: 10000 })
```

Comment:
no blocker, just curious why this one has `{ timeout: 10000 } `but others don't have?

Reply:
It is a remnant from an earlier iteration of the code. I had changed the global default timeout duration from 4 to 10 seconds and deleted the `{ timeout: 10000 }` options for most of the commands, but it looks like I forgot to delete that one. I'll delete it and look for other instances I may have missed removing it.